Files/Views/LayoutModes/GenericFileBrowser.xaml.cs

```diff
@@ -112,8 +112,10 @@ private void SelectionRectangle_SelectionEnded(object sender, EventArgs e)
         protected override void OnNavigatedTo(NavigationEventArgs eventArgs)
         {
             base.OnNavigatedTo(eventArgs);
+            AllView.ItemsSource = ParentShellPageInstance.FilesystemViewModel.FilesAndFolders;
```

Comment:
What benefits will we get on setting ItemsSource in code behind?

Reply:
We set the ItemsSource to null after each navigation because it frees memory unlike simply using Clear(). This is because ListView builds an internal list for caching purposes and setting ItemsSource to null properly clears it. 

> ListView makes the distinction between hard and soft resets.
> A hard reset is when you set the ItemsSource to a new value (or to null). In that case, ListView will throw away all its containers.
> A soft reset is when you keep the same ItemsSource but you clear the collection. In that case, ListView treats it as any other collection change (e.g. add/remove) and put the containers in the recycle pool so that those containers can be reused in the future. Those containers don’t leave the visual tree and FrameworkElement.Unloaded won’t be raised because of that.

Reply:
I mean, you can keep binding ItemsSource in xaml, but set it to null (when needed) in code behind. This PR does both this operations in code behind.

Reply:
@tsvietOK Wouldn't setting it to null break the existing XAML binding?